site2/website/versioned_docs/version-2.1.0-incubating/client-libraries-java.md

```diff
@@ -475,7 +475,7 @@ Pulsar currently supports two authentication schemes: [TLS](security-tls-authent
 
 ### TLS Authentication
 
-To use [TLS](security-tls-authentication), you need to set TLS to `true` using the `setUseTls` method, point your Pulsar client to a TLS cert path, and provide paths to cert and key files.
+To use [TLS](security-tls-authentication.md), you need to set TLS to `true` using the `enableTls` method, point your Pulsar client to a TLS cert path, and provide paths to cert and key files.
```

Comment:
@momo-jun FYI: here is a missing occurrence (the `.md` issue)

Reply:
Thanks for the reminder. I didn't have a chance to fix them in doc versions earlier than 2.10. There are hundreds of them in each version, so I need to implement it separately to avoid getting dizzy and crazy. Maybe I can file an good-first-issue :)